pyqtgraph/Vector.py

```diff
@@ -12,35 +12,37 @@ class Vector(QtGui.QVector3D):
     """Extension of QVector3D which adds a few helpful methods."""
     
     def __init__(self, *args):
+        """
+        Handle additional constructions of a Vector
+
+        ==============  ================================================================================================
+        **Arguments:**
+        *args*      Could be any of:
```

Comment:
Vector isn't currently in the docs, but there should be few more spaces introduced before "Could". It should align with the `=` after the gap in the table borders. Please also add an empty line after this one - I'm not sure exactly why but RST/sphinx seems to prefer it this way for bulleted lists.

Reply:
Thanks; I was bumbling about for the rules there.